### PR TITLE
DYN-7162 Add back missing node help docs and fix cases were not working

### PR DESCRIPTION
**Pull request by QilongTang**
### Purpose

Per Slack [conversation](https://autodesk.slack.com/archives/C186YR8SH/p1718899061111949), team finished a round of internal checking on nodes missing help content. In this PR, I am adding back missing dyns that I could validate which were missing.

Also fixed image display for the following nodes:

- DSCore.Math.Random(value1, value2, seed)
- DSCore.Math.RandomList(amount, value1, value2, seed)
- DSCore.Math.Abs(number)
- DynamoUnits.Quantity.ByTypeID

Added copy step for excel sheets in help docs

- CoreNodeModels.Input.CustomSelection.dyn added by @twastvedt ~~It looks like the dyn file was missing from original PR: https://github.com/DynamoDS/Dynamo/pull/11958)~~

Node with issue

- OpenXML nodes, excel sheets will always come with path problem

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [x] This PR contains no files larger than 50 MB

### Release Notes

Add back missing node help docs so that customer could see in product help

### Reviewers

@DynamoDS/dynamo 

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

**Comment by twastvedt** [comment]
> FYI: @twastvedt It looks like the dyn file was missing from original PR
Added @QilongTang !

**Comment by QilongTang** [comment]
> > FYI: @twastvedt It looks like the dyn file was missing from original PR
> 
> Added, @QilongTang !

Awesome, thanks!

**Comment by QilongTang** [comment]
SelfServe job interrupted, restarted at https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/16229/

File: doc/distrib/NodeHelpFiles/DSCore.Math.Random(value1, value2, seed).md [hunk 1]
```diff
@@ -3,4 +3,4 @@ Math.Random(value1,value2,seed) creates a random number between two values. This
 ___
 ## Example File
 
-![Math.Random2](./DSCore.Math.Random(value1, value2, seed)_img.png)
\ No newline at end of file
+![Math.Random2](./DSCore.Math.Random%28value1%2C%20value2%2C%20seed%29_img.png)
```
Comment (RobertGlobant20) [inline doc/distrib/NodeHelpFiles/DSCore.Math.Random(value1, value2, seed).md:6]: I guess you tested this one and is showing the documentation correctly, right?

**Review by RobertGlobant20 — APPROVED** [review]
LGTM

File: doc/distrib/NodeHelpFiles/DSCore.Math.Random(value1, value2, seed).md (hunk 1 above)
Comment (QilongTang) [reply doc/distrib/NodeHelpFiles/DSCore.Math.Random(value1, value2, seed).md:6]: Yeah, I was using the VSCode md preview and it worked after the change

**Comment by QilongTang** [comment]
It was passing last night on https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/16225/, the CI seems to be down today so merging for now and FYI @sm6srw

**Pull request merged**